tools/importers/helpers/fileReader.ts

```diff
@@ -0,0 +1,65 @@
+import { ITypographyToken } from './structures';
+
+const modifiers = require('./modifiers');
+
+const {
+	fontFamilyProperty,
+	fontWeightKey,
+	transformCssProperty,
+} = require('../helpers/typographyVariables');
+
+export const recursiveTokenReader = (obj, keyResult: string, result: Array<ITypographyToken>) => {
+	if ('value' in obj) {
+		return;
+	}
+
+	for (let key in obj) {
+		let temporaryKey = (keyResult + ' ' + key).replace(/\-+/g, ' ');
```

Comment:
don't we need to skip `deafult` in token name?

Reply:
We need, fixed.